.github/workflows/docker-image-saas-test.yml

```
@@ -0,0 +1,94 @@
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Verify Google Client ID Secret
+        run: |
+          if [ -z "${{ secrets.GOOGLE_CLIENT_ID }}" ]; then
+            echo "GOOGLE_CLIENT_ID secret is NOT available"
+            exit 1
+          fi
+
+      - name: Build and push Clients image
+        uses: docker/build-push-action@v4
```

actionlint (1.7.8)

the runner of "docker/build-push-action@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)